wb/nm_helper/connection_manager.py

```diff
@@ -145,35 +164,57 @@ def wait_device_for_connection(
                     logging.debug("Current device path: {}".format(new_dev_path))
                     if dev_path != new_dev_path:
                         logging.debug('Device path changed from {} to {}'.format(dev_path, new_dev_path))
+                        logging.info('Changed SIM slot to {} to check connectivity'.format(sim_slot))
                         return dev
             except dbus.exceptions.DBusException as ex:
                 # Some exceptions can be raised during waiting, because MM and NM remove and create devices
                 logging.debug("Error during device waiting: %s", ex)
             time.sleep(1)
+        logging.info("Timeout reached while trying to change SIM slot")
         return None
 
+    def can_change_sim_slot(self, dev: NMDevice):
+        active_connection = dev.get_active_connection()
+        if active_connection:
+            logging.debug('Active gsm connection is {}'.format(active_connection))
+            if active_connection.get_connection_type() == 'gsm' and self.deny_sim_switch_until \
+                    and self.deny_sim_switch_until > datetime.datetime.now():
+                logging.info('SIM switch disabled until {}, not changing SIM'.format(
```

Comment:
```suggestion
                logging.debug('SIM switch disabled until {}, not changing SIM'.format(
```

Кажется, будет наваливать в логи иначе